index.html

```diff
@@ -722,8 +722,12 @@ <h2>
             From the perspective of a <a>document</a>, locking to the
             <a>default orientation</a> is equivalent to unlocking because it
             means that it no longer has a lock applied. However, this does not
-            mean that the <a>[[\defaultOrientation]]</a> only contains the item
-            <a>any</a>.
+            mean that the <a>[[\defaultOrientation]]</a> should only contain
```

Comment:
Careful with rfc2119 keyword use here. 
```suggestion
            mean that the <a>[[\defaultOrientation]]</a> will only contain
```